src/Neo.Extensions/BigIntegerExtensions.cs

```
@@ -18,24 +18,34 @@ namespace Neo.Extensions
 {
     public static class BigIntegerExtensions
     {
+
+        internal static int TrailingZeroCount(byte[] b)
+        {
+            var w = 0;
+            while (b[w] == 0) w++;
+            for (var x = 0; x < 8; x++)
+            {
+                if ((b[w] & 1 << x) > 0)
+                    return x + w * 8; // cannot greater than 2Gib
+            }
+            return -1; // unreachable, because returned earlier if value is zero
```

Before returns an Exception, is this method VM accessible?

---

> Before returns an Exception, is this method VM accessible?

This line is unreachable.
The comments explained it.

---

`if (value.Sign == 0) return -1;` before this method.

---

then is better to throw an Exception